packages/forest_admin_agent/lib/forest_admin_agent/routes/capabilities/collections.rb

```diff
@@ -0,0 +1,45 @@
+require 'json'
+module ForestAdminAgent
+  module Routes
+    module Capabilities
+      class Collections < AbstractRoute
```

Comment:
Is this route is accessible without authentication ? Or it's just me reading ruby ?

Reply:
well observed. It's fixed